build_tools/benchmarks/export_benchmark_config.py

```diff
@@ -23,28 +23,75 @@
 # Add build_tools python dir to the search path.
 sys.path.insert(0, str(pathlib.Path(__file__).parent.with_name("python")))
 
+from typing import Callable, Dict, List
 import argparse
 import collections
 import dataclasses
 import json
```

Comment:
Nit: this should be alphabetical

```suggestion
import argparse
import collections
import dataclasses
import json
from typing import Callable, Dict, List
```